package.json

```diff
@@ -3,9 +3,10 @@
     "version": "1.0.0",
     "license": "MIT",
     "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.15.4",
```

Comment:
How come we need to import this package?

Reply:
The flag icon comes from font awesome - https://fontawesome.com/v4.7/icon/flag

Reply:
Ah Jolly good :)